ydb/core/blobstorage/dsproxy/bridge/bridge.cpp

```diff
@@ -0,0 +1,191 @@
+                    self.SelfId().Send(Sender, ev.Release(), 0, Cookie);
+                    Finished = true;
+                }
+            }
+
+            void ProcessResponse(TThis& self, TAutoPtr<TEvBlobStorage::TEvPutResult> ev) {
+                ProcessWritingResponse(self, ev);
+            }
+
+            void ProcessResponse(TThis& self, TAutoPtr<TEvBlobStorage::TEvGetResult> ev) {
+                ProcessReadingResponse(self, ev);
+            }
+
+            void ProcessResponse(TThis& self, TAutoPtr<TEvBlobStorage::TEvBlockResult> ev) {
+                ProcessWritingResponse(self, ev);
+            }
+
+            void ProcessResponse(TThis& self, TAutoPtr<TEvBlobStorage::TEvGetBlockResult> ev) {
+                ProcessReadingResponse(self, ev);
+            }
+
+            void ProcessResponse(TThis& self, TAutoPtr<TEvBlobStorage::TEvDiscoverResult> ev) {
+                ProcessReadingResponse(self, ev);
+            }
+
+            void ProcessResponse(TThis& self, TAutoPtr<TEvBlobStorage::TEvRangeResult> ev) {
+                ProcessReadingResponse(self, ev);
+            }
+
+            void ProcessResponse(TThis& self, TAutoPtr<TEvBlobStorage::TEvCollectGarbageResult> ev) {
+                ProcessWritingResponse(self, ev);
+            }
+
+            void ProcessResponse(TThis& self, TAutoPtr<TEvBlobStorage::TEvStatusResult> ev) {
+                ProcessReadingResponse(self, ev);
+            }
+
+            void ProcessResponse(TThis& self, TAutoPtr<TEvBlobStorage::TEvPatchResult> ev) {
+                ProcessWritingResponse(self, ev);
+            }
+
+            void ProcessResponse(TThis& self, TAutoPtr<TEvBlobStorage::TEvAssimilateResult> ev) {
+                ProcessReadingResponse(self, ev);
+            }
+
+            void ProcessResponse(TThis& self, TAutoPtr<TEvBlobStorage::TEvCheckIntegrityResult> ev) {
+                ProcessReadingResponse(self, ev);
+            }
+        };
+        THashMap<ui64, TRequest> RequestsInFlight;
+        ui64 LastRequestCookie = 0;
+
+    public:
+        TBridgedBlobStorageProxyActor(TIntrusivePtr<TBlobStorageGroupInfo> info)
+            : TActor(&TThis::StateFunc)
+            , Info(std::move(info))
+            , GroupId(Info->GroupID)
+        {}
+
+        template<typename TEvent>
+        void HandleProxyRequest(TAutoPtr<TEventHandle<TEvent>>& ev) {
+            const ui64 cookie = ++LastRequestCookie;
+            const auto [it, inserted] = RequestsInFlight.try_emplace(cookie, ev->Sender, ev->Cookie);
+            Y_ABORT_UNLESS(inserted);
+
+            const auto& bridgeGroupIds = Info->GetBridgeGroupIds();
+            for (size_t i = 0; i < bridgeGroupIds.size(); ++i) {
+                std::unique_ptr<IEventBase> eventToSend(
+                    i + 1 != bridgeGroupIds.size()
+                        ? new TEvent(TEvBlobStorage::CloneEventPolicy, *ev->Get())
+                        : ev->ReleaseBase().Release()
```

Comment:
```suggestion
                    i + 1 == bridgeGroupIds.size()
                        ? ev->ReleaseBase().Release()
                        : new TEvent(TEvBlobStorage::CloneEventPolicy, *ev->Get())
```
it seems easier to understand logic without extra negation